src/control/actor/gossip/topic/export_metris.go

```diff
@@ -6,108 +6,46 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"runtime/debug"
 	"strconv"
 	"time"
 
-	//"github.com/libp2p/go-libp2p-core/host"
+	"github.com/cortze/rumor/p2p/track"
```

Comment:
Apparently my local repo was still set on the old Cortze/armiarma.git repo, which made me not fetch correctly the last commits on the branch.
Sorry for having to post the merge to the repo, didn't really know which was the best way to handle it